cea/optimization/master/summarize_network.py

```diff
@@ -92,169 +74,196 @@ def network_main(locator, total_demand, building_names, config, gv, key):
-                                                                                     HEAT_CAPACITY_OF_WATER_JPERKGK, "negative")
-
-    day_of_max_heatmassflow_fin = np.zeros(HOURS_IN_YEAR)
-    day_of_max_heatmassflow = find_index_of_max(mdot_heat_netw_all_kgpers)
-    day_of_max_heatmassflow_fin[:] = day_of_max_heatmassflow
-
-    for i in range(HOURS_IN_YEAR):
-        if T_DCN_space_cooling_data_center_and_refrigeration_sup_K[i] > T_DCN_space_cooling_data_center_and_refrigeration_re_K[i]:
-            print (i)
-
-        if T_DCN_space_cooling_and_refrigeration_sup_K[i] > T_DCN_space_cooling_and_refrigeration_re_K[i]:
-            print (i)
-
-    date = pd.read_csv(locator.get_demand_results_file(building_names[0])).DATE.values
-    results = pd.DataFrame({"DATE": date,
-                            "mdot_DH_netw_total_kgpers": mdot_heat_netw_all_kgpers,
-                            "mdot_cool_space_cooling_and_refrigeration_netw_all_kgpers": mdot_cool_space_cooling_and_refrigeration_netw_all_kgpers,
-                            "mdot_cool_space_cooling_data_center_and_refrigeration_netw_all_kgpers": mdot_cool_space_cooling_data_center_and_refrigeration_netw_all_kgpers,
-                            "Q_DHNf_W": Q_DHNf_W,
-                            "Q_DCNf_space_cooling_and_refrigeration_W": Q_DCNf_space_cooling_and_refrigeration_W,
-                            "Q_DCNf_space_cooling_data_center_and_refrigeration_W": Q_DCNf_space_cooling_data_center_and_refrigeration_W,
-                            "T_DHNf_re_K": T_DHN_re_K,
-                            "T_DCNf_space_cooling_and_refrigeration_re_K": T_DCN_space_cooling_and_refrigeration_re_K,
-                            "T_DCNf_space_cooling_data_center_and_refrigeration_re_K": T_DCN_space_cooling_data_center_and_refrigeration_re_K,
-                            "T_DHNf_sup_K": T_DHN_sup_K,
-                            "T_DCNf_space_cooling_and_refrigeration_sup_K": T_DCN_space_cooling_and_refrigeration_sup_K,
-                            "T_DCNf_space_cooling_data_center_and_refrigeration_sup_K": T_DCN_space_cooling_data_center_and_refrigeration_sup_K,
-                            "Qcdata_netw_total_kWh": Qcdata_netw_total_kWh,
-                            "day_of_max_heatmassflow": day_of_max_heatmassflow,
-                            "mcpdata_netw_total_kWperC": mcpdata_netw_total_kWperC,
-                            "Electr_netw_total_W": Electr_netw_total_W,
-                            "Q_DH_losses_W": Q_DH_losses_W,
-                            "Q_DC_space_cooling_and_refrigeration_losses_W": Q_DC_space_cooling_and_refrigeration_losses_W,
-                            "Q_DC_space_cooling_data_center_and_refrigeration_losses_W": Q_DC_space_cooling_data_center_and_refrigeration_losses_W})
+
+    #RUN FOR HEATING NETWORKS
+    if network_type == "DH":
+        iteration = 0
```

Comment:
by not extracting the body of this if to a method, it obscures the high-level control flow. also, the `if network_type == "DH"` checks make me nervous because I'm assuming we have a lot of code duplication here - would good to factor out the changing parts but keep the stuff that stays the same...

Reply:
true maybe add a a separate issue for this

Reply:
see #2187